src/main/java/net/yogstation/yogbot/http/GithubEndpoint.kt

```diff
@@ -124,10 +128,14 @@ class GithubEndpoint (
 		}
 	}
 
-	private fun getEventPrEmbed(data: JsonNode, changelog: YogResult<GithubManager.Changelog?, String?>): EmbedCreateSpec {
+	private fun getEventPrEmbed(
+		data: JsonNode,
+		titleOverride: String?,
```

Comment:
Any reason you'd want to override anything but the action? Maybe just take overrideAction for simplicity?

Reply:
For events its "A PR has been $action by $doer"
For pr lookup its "$action Pull Request"

Reply:
But getEventPrEmbed is only used once?